src/callbacks/evaluate_progress.cpp

```diff
@@ -73,6 +74,7 @@ EvalType evaluate(model& m, const std::string& metric_name, execution_mode mode)
                 m.get_name(),
                 "\"");
   }
+  */
```

Comment:
Following on to above suggestion:
```suggestion
  }
```